stable/kong/templates/controller-deployment.yaml

```diff
@@ -62,6 +62,8 @@ spec:
         {{- include "kong.license" . | nindent 8 }}
         {{- end }}
         {{- include "kong.env" .  | indent 8 }}
+        - name: KONG_NGINX_HTTP_INCLUDE
+          value: /kong/servers.conf
```

Comment:
This pod will never proxy traffic and hence exposing metrics on this will not give any insights.
Let's remove this.

Reply:
done